docs/about.md

```diff
@@ -5,21 +5,54 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
+
+
+We are the **[NHS England](https://www.england.nhs.uk/) Data Science Team**. 
+
+**Our vision is**: 
 
 > For NHS England to lead on embracing data science in the NHS for the betterment of all our patients and all our staff​. 
 
-<img src=../images/DS_team_photo.jpeg alt-tex="Picture of the Data Science team stood on some steps in London." width="600" >
-</img>
+
```

Comment:
an extra space between our vision and the contact us button would look better i think, currently theyre a little too squished together.

Reply:
Added!